_includes/components/video.html

```diff
@@ -1 +1 @@
-<video src="{{ include.url }}" width="400" controls=""></video>
+<video src="{{ include.url }}" width="100%" controls=""></video>
```

Comment:
Is this bulletproof?

Reply:
It's as bulletproof as use CSS to set the width to 100%;